tests/integration/features/serverless_feature_test.go

```diff
@@ -351,3 +314,19 @@ var _ = Describe("Serverless feature", func() {
 	})
 
 })
+
+func CreateSubscription(subscriptionYaml, namespace string) {
```

Comment:
I would put it in its own file outside of serverless test as it's used not only here. Perhaps creating `cluster_test_fixtures.go` file under the same package and moving it there is a good enough for start. 

Then next PR can move all the other shared funcs like `newNamespace` to this newly created file as well. WDYT?

Reply:
Just to be sure - under same package means under `integration/features` or are you thinking under `integration/features/fixtures`? I think either could work so want to be on the same page.

Reply:
[9aad31c](https://github.com/opendatahub-io/opendatahub-operator/pull/899/commits/9aad31c8b0444efa7399687b85ace230163a4564)

done now! set up for future PR adding more